src/main/java/org/reactome/qa/doisuggester/Main.java

```diff
@@ -457,17 +412,36 @@ else if (!projectText.equals("Reactome"))
 			// TODO: Iterate through all authors
 			GKInstance personInst = (GKInstance) ie.getAttributeValue(ReactomeJavaConstants.author);
 			String projectText = (String) personInst.getAttributeValue("project");
-			if (projectText == null)
+			if (projectText == null || !projectText.equals("Reactome"))
 			{
 				newNonReactomeInstanceEdits.add(ie);
 			}
-			else if (!projectText.equals("Reactome"))
-			{
-				newNonReactomeInstanceEdits.add(ie);
+		}
+		return newNonReactomeInstanceEdits;
+	}
 
+	private static List<GKInstance> findNewInstanceEdits(GKInstance currentRLE, List<GKInstance> currentRLEInstances, List<GKInstance> previousRLEInstances, String listType)
+	{
+		List<GKInstance> newInstanceEdits = new ArrayList<>();
+		
+		for (int i = 0; i < currentRLEInstances.size(); i++)
+		{
+			GKInstance currentIE = currentRLEInstances.get(i);
+			GKInstance previousIE = null;
+			if (previousRLEInstances.size() > i)
+			{
+				previousIE = previousRLEInstances.get(i);
+				if (!currentIE.getDBID().equals(previousIE.getDBID()))
+				{
+					System.out.println("\'" + listType + "\' elements between current and previous versions of instance do not line up for " + currentRLE.getExtendedDisplayName());
```

Comment:
Could this be logged as a warning or error instead of included with the output?

Reply:
Yes... _WHEN_ a proper logging framework is used in this project. In a future PR.